### DISCUSSION TITLE
Function call and parameter to fill

### DISCUSSION THREAD

**Discussion by mahmoud-abusalameh** [OOlashyn/PCF-AttachmentGalleryControl#25] [Q&A] [answered]
Hello @OOlashyn 

Thank you very much for the solution, 

We added the PCF Control to our environment and added the script to the field (1,2) what do we have to fill in the function call (4) ? and with the parameters (4,5) we did not figure it from the wiki description

![MicrosoftTeams-image (81)](https://user-images.githubusercontent.com/72506137/153628120-612e938b-a44b-41f1-aae0-98729580fa2f.png)

Thank you,
Mahmoud.

**Comment by OOlashyn** [comment, marked as answer]
Hi @mahmoud-abusalameh ,
Please check the Wiki to see how to add component on the form
